ClearlyEnergy/django-helpdesk
# Ticket Forms: Add checkbox list input

#### What's this PR do?
Adds an input that allows users to specify a list of values to be shown in a list of checkboxes in a ticket form.

#### How should this be manually tested?
Edit a form to use the "Checkbox List" data type and add a few values to the list. Go to the ticket submission page and select a few checkboxes in that field and submit. In the created ticket, see the results as a list of values that were checked.

#### What are the relevant tickets?
https://clearlyenergy-inc.monday.com/boards/7029256041/pulses/6079186433

#### Screenshots (if appropriate)
![image](https://github.com/user-attachments/assets/410cb8ba-e8b1-463c-be9c-755bb1204468)
![image](https://github.com/user-attachments/assets/38749e82-e870-495c-80b5-8ff382cdd5eb)
![image](https://github.com/user-attachments/assets/efb9bb5e-4d58-4cea-af72-3a34f52c4b8d)
